### PR TITLE
Add descriptions to children docs-Studio Pro How-tos

### PR DESCRIPTION
**Pull request by Yiyun333**

### CODE REVIEW

File: content/en/docs/howto/data-models/create-a-basic-data-layer.md [hunk 1]
```diff
@@ -3,6 +3,7 @@ title: "Create a Basic Data Layer"
 url: /howto/data-models/create-a-basic-data-layer/
 category: "Data Models"
 weight: 1
+description: "Describes how to create entities and attributes, add enumerations, create associations, and delete association behavior with Studio Pro."
```
Comment (MarkvanMents) [inline content/en/docs/howto/data-models/create-a-basic-data-layer.md]: Is this deleting the behavior or deleting the association? It reads like the former.

File: content/en/docs/howto/general/_index.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 title: "General Info"
 url: /howto/general/
-description: "These introductory how-to's describe Mendix Studio Pro basics and best practices."
+description: "Describes introductory how-tos about Mendix Studio Pro basics and best practices."
```
Comment (MarkvanMents) [inline content/en/docs/howto/general/_index.md]: I think "Presents" works better here. Like you have written in some other descriptions.

File: content/en/docs/howto/integration/_index.md [hunk 3]
```diff
@@ -2,6 +2,7 @@
 title: "Integration"
 url: /howto/integration/
 weight: 70
+description: "Describes a list of how-tos about integrating Mendix applications with other systems and services."
```
Comment (MarkvanMents) [inline content/en/docs/howto/integration/_index.md]:
```suggestion
description: "Presents a list of how-tos about integrating Mendix applications with other systems and services."
```
for consistency

File: content/en/docs/howto/front-end/ux-best-practices.md [hunk 4]
```diff
@@ -3,6 +3,7 @@ title: "Implement Best Practices for UX Design"
 url: /howto/front-end/ux-best-practices/
 category: "Front End"
 weight: 10
+description: "Presents best practices written by a UX Designer for contributing to the app development process."
```
Comment (MarkvanMents) [inline content/en/docs/howto/front-end/ux-best-practices.md]: I think "Describes" is best for single documents, "Presents" is better for a document which is a list of further how-tos. Perhaps you can think about this description, it feels a bit awkward compared to the other ones you have written. Let me know if you want to bounce some ideas around on how to improve it.

image

File: content/en/docs/howto/logic-business-rules/translatable-validation-messages.md [hunk 5]
```diff
@@ -3,6 +3,7 @@ title: "Use Translatable Validation Messages"
 url: /howto/logic-business-rules/translatable-validation-messages/
 category: "Logic & Business Rules"
 weight: 140
+description: "Gives an overview of how to use translatable validation messages."
```
Comment (MarkvanMents) [inline content/en/docs/howto/logic-business-rules/translatable-validation-messages.md]: I like "Describes" for consistency with all the others. How about
```suggestion
description: "Describes the use of translatable validation messages."
```

Or you could begin "Describes how to…" for consistency with the other descriptions.

File: content/en/docs/howto/logic-business-rules/string-concatenation.md [hunk 6]
```diff
@@ -3,6 +3,7 @@ title: "Configure String Concatenation"
 url: /howto/logic-business-rules/string-concatenation/
 category: "Logic & Business Rules"
 weight: 110
+description: "Presents three examples as well as best practices on configuring string concatenation."
```
Comment (MarkvanMents) [inline content/en/docs/howto/logic-business-rules/string-concatenation.md]: Describes...?

File: content/en/docs/howto/integration/implement-cicd-pipeline.md [hunk 7]
```diff
@@ -3,6 +3,7 @@ title: "Implement a Simple CI/CD Pipeline with Mendix APIs"
 linktitle: "CI/CD Pipeline for Mendix Cloud"
 url: /howto/integration/implement-cicd-pipeline/
 category: "Integration"
+description: "Describes how you can use the available Mendix APIs with any mainline orchestrators to build a simple CICD pipeline."
```
Comment (MarkvanMents) [inline content/en/docs/howto/integration/implement-cicd-pipeline.md]: Do we have a standard for CICD or CI/CD. I tend to use the latter, and that is what is in the document title, but I'm not sure we have any guidance about it.

Do you have time to research this a bit in the Microsoft style guide and elsewhere?

File: content/en/docs/howto/monitoring-troubleshooting/clear-warning-messages.md [hunk 8]
```diff
@@ -3,6 +3,7 @@ title: "Clear Warning Messages"
 url: /howto/monitoring-troubleshooting/clear-warning-messages/
 category: "Monitoring & Troubleshooting"
 weight: 1
+description: "Describes how to eliminate design time warnings by viewing and clearing warning messages."
```
Comment (MarkvanMents) [inline content/en/docs/howto/monitoring-troubleshooting/clear-warning-messages.md]: Hyphenated (I think?)
```suggestion
description: "Describes how to eliminate design-time warnings by viewing and clearing warning messages."
```

**Review by MarkvanMents — CHANGES_REQUESTED** [review]
Hi @Yiyun333 ,
This is really great. The site looks so much better when it is built and it will help the customers navigate.
I've added a couple of comments, mainly about trying to be consistent in the use of "Describes" or "Presents". I feel it looks nicer if there is a consistency in what is written - particularly since 90% of the descriptions are consistent.
There are a few docs where the description could also be more consistent with the other descriptions:
/howto/integration/exposing-data-to-bi-tools-using-odata.md
/howto/extensibility/explore-connectors-and-adapters.md
/howto/monitoring-troubleshooting/detect-and-resolve-performance-issues.md

And this file doesn't have a full stop at the end of the description, while all the others do!
/howto/integration/version-rest-service.md

I built the branch locally and found it easier to proofread and to spot inconsistencies. I would never have seen them if I'd tried to work through the pull request.

Let me know if you have any questions or don't like my suggestions.

Thanks for working on this.

Mark

File: content/en/docs/howto/data-models/create-a-basic-data-layer.md (hunk 1 above)
Comment (Yiyun333) [reply content/en/docs/howto/data-models/create-a-basic-data-layer.md]: I changed it to "configure the delete behavior for associations".

File: content/en/docs/howto/general/_index.md (hunk 2 above)
Comment (Yiyun333) [reply content/en/docs/howto/general/_index.md]: Changed it to "presents".

File: content/en/docs/howto/front-end/ux-best-practices.md (hunk 4 above)
Comment (Yiyun333) [reply content/en/docs/howto/front-end/ux-best-practices.md]: Good find! I've changed "presents" to "describes" for single documents.

File: content/en/docs/howto/logic-business-rules/translatable-validation-messages.md (hunk 5 above)
Comment (Yiyun333) [reply content/en/docs/howto/logic-business-rules/translatable-validation-messages.md]: Changed.

File: content/en/docs/howto/logic-business-rules/string-concatenation.md (hunk 6 above)
Comment (Yiyun333) [reply content/en/docs/howto/logic-business-rules/string-concatenation.md]: Sometimes it's not so clear whether to use "describes" and "presents", but I will stick to the rule that for single documents, I will use "describes", and for docs that have a list of other docs, I will use "presents".

File: content/en/docs/howto/integration/implement-cicd-pipeline.md (hunk 7 above)
Comment (Yiyun333) [reply content/en/docs/howto/integration/implement-cicd-pipeline.md]: I will use CI/CD. The Miscrosoft style guide does not have a specific rule for this, but when I search CICD and CI/CD there, it only shows usage of CI/CD, so I guess that's the rule :)

**Comment by Yiyun333** [comment]
> Hi @Yiyun333 , This is really great. The site looks so much better when it is built and it will help the customers navigate. I've added a couple of comments, mainly about trying to be consistent in the use of "Describes" or "Presents". I feel it looks nicer if there is a consistency in what is written - particularly since 90% of the descriptions are consistent. There are a few docs where the description could also be more consistent with the other descriptions: /howto/integration/exposing-data-to-bi-tools-using-odata.md /howto/extensibility/explore-connectors-and-adapters.md /howto/monitoring-troubleshooting/detect-and-resolve-performance-issues.md
> 
> And this file doesn't have a full stop at the end of the description, while all the others do! /howto/integration/version-rest-service.md
> 
> I built the branch locally and found it easier to proofread and to spot inconsistencies. I would never have seen them if I'd tried to work through the pull request.
> 
> Let me know if you have any questions or don't like my suggestions.
> 
> Thanks for working on this.
> 
> Mark

@MarkvanMents Thank you very much for your feedback. I've made changes according to your comments. Please have a look again, and if there are no further issues/suggestions, I will merge this PR.

**Review by MarkvanMents — APPROVED** [review]
All looks good.
Thanks for doing this.

**Pull request merged**